Storage/Processor/Processor.php

```diff
@@ -208,9 +208,9 @@ private function generateImage(Config $config, string $filename = null): string
     }
 
 
-    public function getResource(Config $config, string $filename)
+    public function getResource(Config $config, string $filename, bool $noCache = false)
     {
-        $cacheableResult = $config->cacheableResult();
+        //It's a resource, there is nothing to generate
```

Comment:
In general I believe that comments are hiding code complexity, the code should speak for itself.

Reply:
From the code change I would expect this comment as a commit message?

Reply:
I refactor in order to clarify the cpode (and remove the comment)